src/release.js

```diff
@@ -236,27 +236,32 @@ export default function taoExtensionReleaseFactory(params = {}) {
         async createPullRequest() {
             log.doing('Create the pull request');
 
-            const pullRequest = await githubClient.createReleasePR(
-                data.releasingBranch,
-                params.releaseBranch,
-                data.version,
-                data.lastVersion,
-                subjectType,
-            );
-            if (pullRequest && pullRequest.state === 'open') {
-                data.pr = {
-                    url: pullRequest.html_url,
-                    apiUrl: pullRequest.url,
-                    number: pullRequest.number,
-                    id: pullRequest.id,
-                    full_name: pullRequest.head.repo.full_name,
-                };
-                const labels = ['releases'];
-                await githubClient.addLabel(data.pr.full_name, data.pr.number, labels);
-                log.info(`${data.pr.url} created`);
-                log.done();
-            } else {
-                log.exit('Unable to create the release pull request');
+            try {
+                const pullRequest = await githubClient.createReleasePR(
+                    data.releasingBranch,
+                    params.releaseBranch,
+                    data.version,
+                    data.lastVersion,
+                    subjectType,
+                );
+                if (pullRequest && pullRequest.state === 'open') {
+                    data.pr = {
+                        url: pullRequest.html_url,
+                        apiUrl: pullRequest.url,
+                        number: pullRequest.number,
+                        id: pullRequest.id,
+                        full_name: pullRequest.head.repo.full_name,
+                    };
+                    const labels = ['releases'];
+                    await githubClient.addLabel(data.pr.full_name, data.pr.number, labels);
+                    log.info(`${data.pr.url} created`);
+                    log.done();
+                } else {
+                    log.exit('Unable to create the release pull request');
+                }
+            } catch (err) {
+                log.error(`There are issues preventing the creation of the pull request: ${err}`);
+                log.exit('Please resolve the issues and try again later.');
```

Comment:
It would be a nice place to put a reminder than the release branch needs to be deleted (locally and remotely) before trying the command again. (We could think of generalising it somehow - show it on every unexpected exit - but for now just here would be ok.)

Reply:
That could be done in a step also if it failed after the branch was created, but I suppose that's another type of work :smiling_face_with_tear: but sure, I'll add the deletion of branches as a hint. Good call!